test/business-sync-shoebox-item-spec.js

```diff
@@ -17,7 +17,39 @@ describe('business-sync shoebox item document definition', function() {
       source: 'yodlee',
       sourceId: '1239004',
       received: '2016-06-18T18:57:35.328-08:00',
-      data: '{ "bank-rec": "data" }',
+      data: {
+        foo: 'bar'
+      },
+      previousData: [
```

Comment:
This may be nit picking about test data so feel free to overrule me.

I don't think we're doing anything sophisticated enough with our Synctos validation yet to enforce this and we're still going to be working on the logic to to replace/update a document but I'm wondering if we should update this to be more accurate of what we'd expect.

I would think we might have the value for `foo` iterate from `bar` to `baz` to `qux`? And we'd want the timestamp to change too?

Or perhaps that was the point, that later on when we're dealing with an update that is sent more than once the data would remain the same and this test would highlight that the same data document should not be in the system?

Reply:
Hmm this project only specifies the document structure, it does not enforce the semantics. Although we want increasing timestamps and values to only be written if the document differs, this isn't something we can write easily in synctos, and i think we'll have to test for it in the apps that write to the shoebox.

Reply:
I'm okay with that. We were just discussing validation via synctos and agreed there may be more validation that we want but will have to be done at an application level.